src/clr-core/badge/badge.element.ts

```diff
@@ -40,6 +46,14 @@ import { styles } from './badge.element.css';
  */
 // @dynamic
 export class CwcBadge extends LitElement {
+  /** Sets the color of the badge from a predefined list of choices */
+  @property({ type: String, reflect: true })
+  color: 'gray' | 'purple' | 'blue' | 'orange' | 'light-blue';
+
+  /** Sets the color of the badge from a predefined list of statuses */
+  @property({ type: String, reflect: true })
+  status: 'info' | 'success' | 'warning' | 'danger';
```

Comment:
Not going to block on this. But it seems like we could turn `'info' | 'success' | 'warning' | 'danger'` into its own type. We use it everywhere...

Reply:
Good point, I can add this to my todo list when I update the button from link to flat. I'll open a separate PR to fix both.